src/org/labkey/test/components/ui/entities/EntityBulkUpdateDialog.java

```diff
@@ -286,7 +287,7 @@ public Input numericInput(String fieldKey)
         public ReactDateTimePicker dateInput(String fieldKey)
         {
             return new ReactDateTimePicker.ReactDateTimeInputFinder(getDriver())
-                    .withInputId(fieldKey).waitFor(formRow(fieldKey));
+                    .withInputId(QueryKey.encodePart(fieldKey)).waitFor(formRow(fieldKey));
```

Comment:
Avoid using server API. It can cause trouble. We have a test equivalent of this method.
```suggestion
                    .withInputId(EscapeUtil.fieldKeyEncodePart(fieldKey)).waitFor(formRow(fieldKey));
```

Reply:
If using `EscapeUtil.fieldKeyEncodePart` is preferred on the test side, I'm happy to change to this. Done.